src/build.d

```diff
@@ -479,7 +479,7 @@ void parseEnvironment()
 
     env.getDefault("GIT_HOME", "https://github.com/dlang");
     env.getDefault("SYSCONFDIR", "/etc");
-    env.getDefault("TMP", "/tmp");
+    env.getDefault("TMP", detectTemp);
```

Comment:
How about: https://dlang.org/phobos/std_file#tempDir ?

Reply:
Good point, we’re already using Phobos.